analysis_options.yaml

```diff
@@ -81,7 +81,6 @@ linter:
     - cancel_subscriptions
     # - cascade_invocations # not yet tested
     - cast_nullable_to_non_nullable
-    - collection_methods_unrelated_type
```

Comment:
why is this removed? if this is a one of case, we should probably use // ignore ?

Reply:
Huh.  I removed it because I was getting an analyzer warning that it wasn't a valid lint anymore, but now that I put it back, there's no warning.